platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/state/SignatureTransmitter.java

```diff
@@ -58,20 +58,11 @@ public SignatureTransmitter(
      * 		the hash of the state that was signed
      */
     public void transmitSignature(final long round, final Signature signature, final Hash stateHash) {
-        if (isZeroStake()) {
```

Comment:
It does not make sense to remove this check. If the node has no consensus weight, there is no point in sending a state signature transaction. It will not help other nodes to gain more confidence about the network reaching the same state.

Reply:
One marginal benefit of transmitting the signature is that the signature transaction is used to detect ISSes. Without sending the signature transaction, this node will still realize when it has an ISS with respect to the rest of the network, but the rest of the network won't observe the zero stake node entering an ISS state. Not a huge deal, but will reduce debugability slightly.

If we do decide to start transmitting these signatures, then we definitely should not add them to a `SignedState`. When attached to a signed state, a zero weight signature is not worth anything. The place where we would want to enforce this is `SignedState.addSignature()`.

Reply:
Adding check to `SignedState.isSignatureValid()` to return false if the address has zero stake.   This also throws away the signature in `SignedState.pruneInvalidSignatures()`

Adding UnitTest to `StateSigningTests.java` for both scenarios.